addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -238,4 +239,24 @@ options.registry.productsSearchBar = options.Class.extend({
         }).open();
     },
 });
+
+sAnimations.registry.websiteSaleCurrency = sAnimations.Class.extend({
+    selector: '.oe_website_sale',
+    read_events: {
+        'click .oe_currency_value:o_editable': '_onCurrencyValueClick',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onCurrencyValueClick: function (ev) {
+        if (this.editableMode) {
+            this.$(ev.currentTarget).selectContent();
```

Comment:
@kea14 Also, no `this.` here. You are building a jQuery element, not searching the DOM